pumpkin-cli/src/main.rs

```diff
@@ -375,7 +377,9 @@ fn configure_logging_unknown() -> std::io::Result<()> {
 }
 
 fn configure_logging_minizinc(verbose: bool, log_statistics: bool) -> std::io::Result<()> {
-    pumpkin_lib::statistics::configure(log_statistics, "%%%mzn-stat:", Some("%%%mzn-stat-end"));
+    if log_statistics {
```

Comment:
It is probably easier to do this check in `run` (`main.rs`) where `configure_logging` is called.

Reply:
I think the problem there would be that we can consider logging without logging statistics (e.g. for printing comments) which we would still want to set up using the normal workflow (if I've understood your comment correctly!)